docs/configuration/outbox-event-router.asciidoc

```diff
@@ -184,10 +186,11 @@ After observing all those pieces we can see what the default configuration does:
 |Becomes the Kafka message key, which is important for keeping ordering within Kafka partitions
 
 |`type`
-|The `type` column becomes part of the message envelope as `eventType`
+|The `type` column becomes either part of the message envelope as `eventType` or message header. +
+See option `table.fields.additional.placement` for more details.
 
 |`payload`
-|The JSON representation of the event itself, becomes part of the message as `payload`
+|The JSON representation of the event itself, becomes either part of the message as `payload` or if other metadata including `eventType` are delivered as headers then the payload becomes the message itself without an encapsulation in an envelope
```

Comment:
Maybe also mention this behavior is attached to the additional fields one?

Reply:
@renatomefi I ma not sure I understand your suggestion, could you please elaborat more or propose the update, thanks!